RSDK-2062 Document build and run steps (#2)
viam-labs/wifi-sensor: README.md

# wifi-sensor

A sensor that detects wifi strength

## Usage

### 1. Build binary

If you clone this repository to the target environment where you run your Viam robot, then you can build a binary named `wifi` with:

```
go build -o wifi
```

> Alternatively, if you want to build this a binary for a different target environment, please use the [viam canon tool](https://github.com/viamrobotics/canon).

Reviewer: ```suggestion
Alternatively, if you want to build a binary for a different target environment, please use the [viam canon tool](https://github.com/viamrobotics/canon).
```

### 2. Add to robot configuration

> Copy the binary to in the environment where your Viam robot is running and add the following to your configuration:

Reviewer: ```suggestion
Copy the binary to the robot (system where viam-server is running) and add the following to your configuration:
```

```
  ...
  "modules": [
    ...,
    {
      "executable_path": "<path_to_binary>",
      "name": "wifi"
    }
    ...,
  ],

>   ...

Reviewer: I would also add the config for the wifi unit itself here. The part that would go in the "components" section.

# wifi-sensor

A sensor that detects wifi strength

## Usage

### 1. Build binary

If you clone this repository to the target environment where you run your Viam robot, then you can build a binary named `wifi` with:

```
go build -o wifi
```

Alternatively, if you want to build a binary for a different target environment, please use the [viam canon tool](https://github.com/viamrobotics/canon).

### 2. Add to robot configuration

Copy the binary to the robot (system where viam-server is running) and add the following to your configuration:

```
  ...
  "modules": [
    ...,
    {
      "executable_path": "<path_to_binary>",
      "name": "wifisensor"
    },
    ...,
  ],
  "components": [
    ...,
    {
      "name": "wifi",
      "type": "sensor",
      "model": "viam-labs:sensor:linux-wifi",
      "depends_on": [],

>       "attributes": {}

Reviewer: [nit] I think these could just be left out entirely, but no harm with them here either.

>       "model": "viam-labs:sensor:linux-wifi",
>       "depends_on": [],
>       "attributes": {}

Author: ```suggestion
      "model": "viam-labs:sensor:linux-wifi"
```
